### PR TITLE
Implementation and Updating Submodules

### PR DESCRIPTION
**Pull request by piepie62**
I have updated the submodules and done my best at implementing new and older methods.

### CODE REVIEW

File: src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java [hunk 1]
```diff
@@ -32,4 +34,14 @@ public SpongeHorseColor(int color, String name) {
         super(color, name);
     }
 
+    @Override
+    public DataContainer toContainer() {
+        return (DataContainer)this;
```
Comment (gabizou) [inline src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java]: 1) [`DataSerializable#toContainer()](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/service/persistence/DataSerializable.java#L38) is supposed to return a **NEW** DataContainer that represents the data of the serializable object, not casting the object as a DataContainer.

2) This cast will fail as none of the extending classes implement DataContainer.

Comment (piepie62) [inline src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java]: Okay, this is the one method that I am unsure about.

Comment (piepie62) [inline src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java]: Should I reduce it to a method stub in a new commit?

**Comment by piepie62** [comment]
Okay, I didn't see PR 107, this is unneccesary. Closing.

**Pull request closed without merging**